src/App.jsx

```diff
@@ -0,0 +1,85 @@
+import contactsData from './contacts.json'
+import { useState} from 'react'
+import "./App.css";
+
+function App() {
+  const [contacts, setContacts] = useState(contactsData.slice(0,5))
+ 
```

Comment:
Here is the initial state of remainingContacts 
```suggestion
  const remainingContacts = contactsData.filter((comingContact) => !contacts.some((contact) => comingContact.id == contact.id))
```